.chocolatey/tools/ChocolateyBeforeModify.ps1

```diff
@@ -0,0 +1,11 @@
+$InstallDir = Join-Path $env:APPDATA 'PresenceLight'

```

Comment:
If you move forward with this, this doesn't take into account the ability to configure that installation directory via a Package Parameter, which is done here:

https://github.com/isaacrlevin/presencelight/blob/main/.chocolatey/tools/ChocolateyInstall.ps1#L41